src/test/java/io/github/christophermanahan/carnitas/HTTPRequestTest.java

```diff
@@ -0,0 +1,20 @@
+package io.github.christophermanahan.carnitas;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class HTTPRequestTest {
+    @Test
+    void getsRequestMethod() {
+        RequestMethod requestMethod = RequestMethod.GET;
+        Assertions.assertEquals(requestMethod, new HTTPRequest(requestMethod, null).requestMethod());
```

Comment:
I tend to avoid using `null`, even if it is in a test. I prefer to use something that resembles production data. `null` does it give me any context.